packages/integrations/netlify/src/netlify-edge-functions.ts

```diff
@@ -6,7 +6,7 @@ const clientAddressSymbol = Symbol.for('astro.clientAddress');
 export function createExports(manifest: SSRManifest) {
 	const app = new App(manifest);
 
-	const handler = async (request: Request): Promise<Response | void> => {
+	const handler = async (request: Request, connInfo?: any): Promise<Response | void> => {
```

Comment:
nit: is there a `handler` type we could import from Netlify here instead of using `any`?

Reply:
done, thanks for the suggestion